yoojin_lee/LV_1/2016년.js

```diff
@@ -0,0 +1,7 @@
+function solution(a, b) {
+  let day = new Date(2016, a - 1, b);
+  const WEEKDAY = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"];
+  return WEEKDAY[day.getDay()];
+}
+// getDaty() 메서드는 주어진 날짜의 현지 시간 기준 요일을 반환 (0 - 일요일)
+// 반환 값 : 0 (일요일) ~ 6 정수
```

Comment:
```suggestion
function solution(a, b) {
  let day = new Date(2016, a - 1, b);
  const WEEKDAY = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"];
  return WEEKDAY[day.getDay()];
}
// getDay() 메서드는 주어진 날짜의 현지 시간 기준 요일을 반환 (0 - 일요일)
// 반환 값 : 0 (일요일) ~ 6 정수
```
꼼꼼한 주석이 적혀있는 코드 풀이 잘봤습니다!!